### PR TITLE
Correct handling of creation timestamp

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Lib is https://github.com/nextcloud/android-library/pull/827

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

- [x] Tests written, or not not needed

### CODE REVIEW

File: src/main/java/com/owncloud/android/utils/FileUtil.java [hunk 1]
```diff
@@ -52,4 +60,20 @@ String getFilenameFromPathString(@Nullable String filePath) {
...
+        try {
+            return Files.readAttributes(file.toPath(), BasicFileAttributes.class)
+                .creationTime()
+                .to(TimeUnit.SECONDS);
+        } catch (IOException e) {
+            Log_OC.e(UploadFileRemoteOperation.class.getSimpleName(),
+                     "Failed to read creation timestamp for file: " + file.getName());
+            return -1L;
+        }
+    }
```
Comment (AlvaroBrey) [inline src/main/java/com/owncloud/android/utils/FileUtil.java:78]: What do `1L` and `-1L` mean for this?

**Review by AlvaroBrey — APPROVED** [review]

File: src/main/java/com/owncloud/android/utils/FileUtil.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/utils/FileUtil.java:78]: Uh. Both need to be -1L, which is undefined.
Maybe it should be null instead, to make it more visible?

Comment (AlvaroBrey) [reply src/main/java/com/owncloud/android/utils/FileUtil.java:78]: `null` is more understandable in my opinion. If you go with `-1L`, then please make a constant with a name like `TIMESTAMP_UNDEFINED` or something

Comment (tobiasKaminsky) [reply src/main/java/com/owncloud/android/utils/FileUtil.java]: Updated 👍

**Pull request merged**